bin/lodestar

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env node
+
+require('../src/index.ts');
```

Comment:
We'll want to require files from `lib/` since node can't run TS natively. (So we'll need to run `npm run build` before we can run lodestar)

Reply:
Yupppp, that makes a whole lot of sense.